_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,493 @@
+---
+id: be4d0c
+name: Inline link is distinguishable
+rule_type: composite
+description: |
+  This rule checks that inline links have a way to distinguish them from any surrounding text other than color alone.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_rules:
+  - 88407d
+  - 548868
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
+- **different color**: the element's [foreground color][] and the [foreground color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1, or the element's [background color][] and the [background color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1.
```

Comment:
Not sure why, if you leave color out, it would fail 1.3.1. You wouldn't have any presentation cues to apply 1.3.1 to, if I correctly understood what you're saying.

Reply:
Because I always believe that 1.3.1 goes both ways 😞 

---
Still, if there is no visual cues at all (not even colour) to distinguish an inline link, shouldn't it fails some WCAG SC? It feels weird that having a blue link in black text is bad (indicated only by colour) but a black link in black text is OK (no indication at all)…

Reply:
I would fail SC 1.4.11. A link (which is a UI component) should be visually identifiable from the surrounding colors. Then we have 1.4.1 to make sure that color is not the only means to provide such visual identification.